etc/amazon-eks-vpc.yaml

```diff
@@ -0,0 +1,167 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Amazon EKS Sample VPC'
+
+Parameters:
+
+  VpcBlock:
+    Type: String
+    Default: 192.168.0.0/16
```

Comment:
How does one decide on the IP address and the /16 or /18?

Reply:
A simple comment on this file covering the following would be great.
Can these parameter values be used unchanged in another EKS environment? Or any values need to be changed?

Reply:
Yes that's going to depend on the customer environment. Yes it can be left unchanged and it will create the VPC with the default values.